a3p-integration/.yarn/patches/@agoric-synthetic-chain-npm-0.1.1-d26a740f05.patch

```diff
@@ -0,0 +1,126 @@
+diff --git a/dist/cli/cli.js b/dist/cli/cli.js
+index c59a54224c101648c5507f451bfe0b2cb7f420a5..24ef27a882805df4a969a96ab28dad15b8ee5479 100755
+--- a/dist/cli/cli.js
++++ b/dist/cli/cli.js
+@@ -14,7 +14,9 @@ import path from "node:path";
+ var defaultConfig = {
+   // Tag of the agoric-3 image containing all passed proposals
+   // Must match the Bake file and CI config
+-  fromTag: "latest"
++  fromTag: "latest",
++  repositoryColon: "ghcr.io/agoric/agoric-3-proposals:",
```

Comment:
why is the colon necessary to include?

Reply:
given there are multiple repositories in play, both git and Docker, please give this a more descriptive name. assuming we can drop the colon,

```suggestion
+  a3pDockerRepository: "ghcr.io/agoric/agoric-3-proposals",
```